.github/workflows/build.yml

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2021 Valve Corporation
+# Copyright (c) 2021 LunarG, Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: Lenny Komow <lenny@lunarg.com>
+
+name: CI Build
+
+on:
+    push:
+    pull_request:
+        branches:
+            - master
```

Comment:
I'm probably not reading this right, but I'll make some guesses...

I would have thought that we'd want to run on a push to `master` and any pull request.  This looks like it runs on any push and only pull requests to `master`... is that right?

Reply:
You're correct about what this does. As for why, by running it on any push, it'll run for SDK branches, which I think is a good thing. We mostly use forks for development in this repo, so I don't think running it on any push will trigger all that many jobs.